decidim-initiatives/app/controllers/decidim/initiatives/authorization_create_modals_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Initiatives
+    class AuthorizationCreateModalsController < Decidim::Initiatives::ApplicationController
```

Comment:
Also I was wondering why do we have special routes for this instead of displaying the default authorization modal?

Reply:
And same here, it's the same as the sign one, and maybe it's because it's connected to a special authorization handler